File: 다영/03.반복문/src/1110.py
n = input()
origin_n = n

i = 0
while True:
    if int(n) < 10:
        print(int(n))
        n += '0'

    new_n = str(int(n[0]) + int(n[1]))

    if int(new_n) > 9:
        n = n[1] + new_n[1]
    print(n)

    if n == origin_n:
        break
    n = new_n
    i += 1

print(i)

File: 다영/03.반복문/src/1110.js
// const fs = require('fs')
// let input = fs.readFileSync('/dev/stdin').toString().split(' ')

// let n = input[0]
let n = '26'
let origin_n = n

let result = 0
console.log('start')
while (true) {
    if (Number(n) < 10 ){
        n += '0'
    }

    let new_n = n.split('')[1] + (Number(n.split('')[0]) + Number(n.split('')[1]))

    if (new_n == origin_n){
        break;
    }
    n = String(new_n)
    result++
}

console.log(result)
